tethys_gizmos/views/gizmos/jobs_table.py

```diff
@@ -1,23 +1,43 @@
+import inspect
 import logging
 import re
+from functools import wraps
 
 from django.http import JsonResponse
 from django.template.loader import render_to_string
 from tethys_compute.models import TethysJob, CondorWorkflow, DaskJob, DaskScheduler
 from tethys_gizmos.gizmo_options.jobs_table import JobsTable
 from tethys_sdk.gizmos import SelectInput
 from tethys_portal.optional_dependencies import optional_import
+from django.contrib.auth.decorators import login_required
```

Comment:
Remind me why we aren’t using our `login_required` decorator? It is the one that automatically makes it open in open-portal mode. I think we talked about this, but I need my memory jogged.

Reply:
Yes, I didn't want an open-portal mode to expose the ability for anyone to delete all the jobs in the DB.